index.html

```diff
@@ -16,5 +16,6 @@
 </head>
 <body>
   <script src="sketch.js"></script>
+  <p>Welcome to Angru bird Game</p>

```

Comment:
```suggestion
  <p>Welcome to Angru bird Game</p>
```

Change it as Angry bird